### ISSUE TITLE
Eclipse IDE code style formatter: how to disable ?

### ISSUE THREAD

**Issue by billh0420** [digama0/mmj2#54]
I couldn't find a way to disable Eclipse IDE code style formatter. Can it be done and, if so, how?

**Comment by digama0** [comment]
https://superuser.com/questions/309955/disable-formatting-in-eclipse

**Comment by digama0** [comment]
This is not an mmj2 issue. Please refrain from using the issue tracker for things other than bug reports and pull requests.

**Comment by billh0420** [comment]
I understand that it is not directly a mmj2 issue. But my intent is not to report bugs but to fix bugs.

But I cannot fix bugs and do a pull request if my updates cause unwanted changes caused by reformatting caused by code style requirements. You said in reply to another request that I had that I should turn off code style formatting. I am fine with that.
I did research on it and I found things like the link https://superuser.com/questions/309955/disable-formatting-in-eclipse <https://superuser.com/questions/309955/disable-formatting-in-eclipse> that you posted. But, doing that change for “Save actions” didn’t work for me. I may not have done it correctly. I will look further into it.
